bindings/pyroot/pythonizations/python/ROOT/_pythonization/_roofit/_rooglobalfunc.py

```diff
@@ -256,3 +256,29 @@ def MarkerStyle(style):
     from cppyy.gbl import RooFit
 
     return RooFit._MarkerStyle(_string_to_root_attribute(style, {}))
+
+
+@cpp_signature("DataError(Int_t) ;")
+def DataError(etype):
```

Comment:
Looks good! It would be good to have this new addition in the Doxygen docs of this file, right? Besides modifying the tutorials.

Reply:
Where is a `DataError="SumW2"`-like example shown in the docs, sorry?

Reply:
It's a few lines below the line you made this comment for

Reply:
Saw it now, thanks!